### PR TITLE
Semantic search documentation

### PR DESCRIPTION
**Pull request by andhesmi**

### CODE REVIEW

File: SemanticSearch.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+-[Query Response](#query-response)
+
+-[Document Description](#document-description)
+
+-[Document Request Information](#document-request-information)
+
+-[Document Response](#document-response)
+
+-[Versioning](#versioning)
+
```
Comment (glpayson) [inline SemanticSearch.md:28]: Need a space between the '-' and the '[' to make it a bullet.

File: SemanticSearch.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    "related_keywords_recruiter": "RelatedSearchTermsRecruiterV1",
+    "related_keywords_jobseeker": "RelatedSearchTermsJobSeekerV1",
+    "raw_job_titles_jobseeker": "RawJobTitlesJobSeekerV1",
+    "raw_job_titles_recruiter": "RawJobTitlesRecruiterV1"
+    "job_level": "JobLevel"
+    "text_kernel_related_keywords": "TextKernelRelatedSearchTerms"
+  }
+}
+```
+
```
Comment (glpayson) [inline SemanticSearch.md:211]: Should we maybe describe the difference between the extracted keywords enrichments and the summary enrichments?

File: SemanticSearch.md [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+#Document Description
+
+Used to parse a job or CV/resume into its most important features and to represent these as prioritized list of features that can be used to form a matching query.
```
Comment (matthuhman) [inline SemanticSearch.md]: - "a prioritized list" or "as prioritized lists"

File: SemanticSearch.md (hunk 2 above)
Comment (andhesmi) [reply SemanticSearch.md:211]: I added a small paragraph on both document and query responses to describe this and the Parsed Input node.

**Comment by matthuhman** [comment]
👍

**Comment by thomasmahoney** [comment]
I think the documentation here should make it more clear that "Query" and "Document" are two separate endpoints. It's hard to immediately tell that currently. Maybe something like this for the table of contents:

Semantic Search API

(body text) The semantic search API supports two endpoints: **/query** and **/document**. The /query endpoint [...]. The /document endpoint [...].

(h2) Query endpoint
(link) Description
(link) Request information
(link) Response
(h2) Document endpoint
(link) Description
(link) Request information
(link) Response
(h2) General information
(link) Versioning

**Comment by glpayson** [comment]
:+1:

**Comment by matthuhman** [comment]
👍

**Pull request merged**